src/main/java/org/jabref/gui/collab/ChangeScanner.java

```diff
@@ -94,17 +94,21 @@ public void displayResult(final DisplayResultCallback fup) {
             });
 
         } else {
-            JOptionPane.showMessageDialog(null, Localization.lang("No actual changes found."),
-                    Localization.lang("External changes"), JOptionPane.INFORMATION_MESSAGE);
+            frame.getDialogService().showInformationDialogAndWait(Localization.lang("External changes"),
+                    Localization.lang("No actual changes found."));
+
             fup.scanResultsResolved(true);
         }
     }
 
     private void storeTempDatabase() {
         JabRefExecutorService.INSTANCE.execute(() -> {
             try {
-                SavePreferences prefs = SavePreferences.loadForSaveFromPreferences(Globals.prefs).withMakeBackup(false)
-                        .withEncoding(panel.getBibDatabaseContext().getMetaData().getEncoding()
+                SavePreferences prefs = SavePreferences.loadForSaveFromPreferences(Globals.prefs)
```

Comment:
The code style settings still seem to be off. The dots are not aligned (if this is not an artifact of the way github displays the code)